trade-finance-manager-ui/component-tests/assertions.ts

```diff
@@ -132,4 +132,9 @@ export const assertions = <TParams extends object>(wrapper: CheerioAPI, html: st
       expect(wrapper(selector).attr('aria-sort')).toEqual(text);
     },
   }),
+  expectPageTitle: () => ({
+    toRead: (text: string) => {
+      expect(wrapper('title').text().replace(', Trade Finance Manager, gov.uk', '').trim()).toEqual(text);
```

Comment:
Adding in the `replace` allows us to not repeat the given string (which is added in the nunjucks index template) in each call to this assertion.

Alternatively we could change this to be a `contains` assertion.

Open to thoughts

Reply:
Strict `toEqual` is better, contains is risky

Could we create a `result` const please? Easier to read IMO

```ts
expect(result).toEqual(text);
```

Reply:
Pulled this out thanks